codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/customization/S3ErrorWith200Status.java

```diff
@@ -0,0 +1,72 @@
+package software.amazon.smithy.aws.go.codegen.customization;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import software.amazon.smithy.aws.traits.ServiceTrait;
+import software.amazon.smithy.codegen.core.SymbolProvider;
+import software.amazon.smithy.go.codegen.GoDelegator;
+import software.amazon.smithy.go.codegen.GoSettings;
+import software.amazon.smithy.go.codegen.GoWriter;
+import software.amazon.smithy.go.codegen.SymbolUtils;
+import software.amazon.smithy.go.codegen.integration.GoIntegration;
+import software.amazon.smithy.go.codegen.integration.MiddlewareRegistrar;
+import software.amazon.smithy.go.codegen.integration.RuntimeClientPlugin;
+import software.amazon.smithy.model.Model;
+import software.amazon.smithy.model.shapes.OperationShape;
+import software.amazon.smithy.model.shapes.ServiceShape;
+import software.amazon.smithy.utils.ListUtils;
+
+/**
+ * Adds middleware to handle S3 response errors with 200 ok status code.
+ */
+public class S3ErrorWith200Status implements GoIntegration {
+    private static String ADD_ERROR_HANDLER_INTERNAL = "HandleResponseErrorWith200Status";
+
+    // list of operations for which this customization is valid.
+    private static Set<String> customizedOperations = Stream.of(
```

Comment:
`SetUtils.of`

Reply:
Isn't that a Java9 thing? Thought we were limited to Java8